tools/hangar/__snapshots__/test_corpus/valid/nil.test.w_compile_tf-aws.md

```diff
@@ -12,8 +12,8 @@ module.exports = function({ $foo }) {
       return $obj;
     }
     async handle() {
-      $helpers.assert($helpers.eq((((await $foo.returnNil(true))) != null), true), "foo.returnNil(true)? == true");
```

Comment:
Interesting that `?` checked against `null` when `nil is `undefined` in wing, I wonder if we were relying on that behavior anywhere 🤔

Reply:
I'll keep my eyes peeled